packages/flutter_tools/test/ios/cocoapods_test.dart

```diff
@@ -46,7 +46,7 @@ void main() {
     mockProcessManager = new MockProcessManager();
     mockXcodeProjectInterpreter = new MockXcodeProjectInterpreter();
     projectUnderTest = await FlutterProject.fromDirectory(fs.directory('project'));
-    projectUnderTest.ios.directory.createSync(recursive: true);
+    projectUnderTest.ios.directory..childDirectory('Runner.xcodeproj').createSync(recursive: true);
```

Comment:
`..`, why does this work?

Reply:
Because
```
a..b().c()
```
calls `c` on the return value of `b`, not on the value of `a`. However, the overall expression has the value of `a`. I'll remove one `.`.